ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -57,11 +57,23 @@ open class TwoLineTitleView: UIView {
         case dark
     }
 
+    @objc(MSFTwoLineTitleViewAlignment)
+    public enum Alignment: Int {
+        case center
+        case leading
+    }
+
     @objc(MSFTwoLineTitleViewInteractivePart)
     public enum InteractivePart: Int {
-        case none
-        case title
-        case subtitle
+        case none = 0
+        case title = 1
+        case subtitle = 2
+        case all = 3
```

Comment:
Is all for both title & subtitle? If so, is that any different from subtitle since we typically have a subtitle with a title?